.github/workflows/health-dashboard.yml

```diff
@@ -20,7 +15,7 @@ jobs:
             title: 'Health Dashboard'
             output:
               format: html
-              filename: 'docs/health-dashboard.html'
+              filename: 'GitHubPages/health-dashboard.html'
```

Comment:
Please see if it needs to go inside a folder? For having url path like - azure.github.io/actions/health

Reply:
As per this the url will be azure.github.io/actions/health-dashboard